github-org-owners: clarify low-impact github changes (#55)
nix-open-org/org-v2: doc/github-org-owners.md

  - Adding new org members to allow review requests.
  - Creating new unprivileged Nixpkgs teams for mention.
  - Updating repository meta information.

>   - Adding/Modifying github apps/settings that repository owners can't do by themself and have no impact outside that repository.

Reviewer: I think that's a bit too generic to not need approval from another org owner.

Reviewer: Ah I see now, that makes a lot of sense, I'd word it like this to make it clearer. Notably not nesting under "low-impact changes" anymore, but the thing that makes it okay is having approval from the repository admin, therefore effectively being approved by two people (repo admin + org owner):

```suggestion
- Changes requested by a majority of repository admins, that have no impact outside their repository.
```

(and edited to make sure this doesn't break when there's multiple repo admins)

Reviewer 2: Infra is certainly not going to hold majority votes among 7 [infra-build team](https://github.com/orgs/NixOS/teams/infra-build) members, where only some are actively reading the infra team chat. :grimacing:

Reviewer: ```suggestion
- Changes requested by a repository admin that have no impact outside their repository.
```

Good point, I think we can assume that repository admins only make requests once the local decision-making process (if any) has been followed.

(lol alternative: Admins need to remove all other admins so that a single person is a majority, then make the request, then re-add other admins after it was granted)
